tests/ui/test_ui.py

```diff
@@ -266,21 +266,22 @@ def test_keypress_ALL_MENTIONS(self, view, mocker, key, widget_size):
 
     @pytest.mark.parametrize("key", keys_for_command("STREAM_MESSAGE"))
     def test_keypress_STREAM_MESSAGE(self, view, mocker, key, widget_size):
-        view.middle_column = mocker.Mock()
+        mocker.patch(VIEW + ".middle_column", create=True)
+        mocked_middle_column_keypress = mocker.patch(VIEW + ".middle_column.keypress")
```

Comment:
Could we simplify this to use:
* mocker.patch.object on view with "middle_column"? (create=True still probably necessary?)
* assert on that object .keypress?

Same for the users_view case.